website/docs/d/cs_cluster_credential.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Container Service for Kubernetes (ACK)"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cs_cluster_credential"
+sidebar_current: "docs-alicloud-datasource-cs-cluster-credential"
+description: |-
+Provides a list of Container Service Cluster's credential to be used by the alicloud_cs_cluster_credential resource.
+---
+
+# alicloud\_cs\_cluster\_credential
+
+This data source provides Container Service cluster credential on Alibaba Cloud.
+
+-> **NOTE:** Available in v1.186.0+
+
+-> **NOTE:** This datasource could be used to ACK clusters, including managed cluster, kubernetes cluster, serverless cluster and edge cluster. Please make sure that the target cluster is not in the failed state before using this datasource, because the cluster in the failed state cannot access the apiserver.
```

Comment:
Suggestion:

```
-> **NOTE:** This datasource can be used on all kinds of ACK clusters, including managed clusters, imported kubernetes clusters, serverless clusters and edge clusters. Please make sure that the target cluster is not in the failed state before using this datasource, since the api server of clusters in the failed state cannot be accessed.
```